frontend/src/components/GlobalStyles.js

```diff
@@ -0,0 +1,7 @@
+import { createGlobalStyle } from 'styled-components';
+export const GlobalStyles = createGlobalStyle`
+  body {
+    background: ${({ theme }) => theme.background};
+    color: ${({ theme }) => theme.text[0]};    
+    transition: all 0.2s linear;
```

Comment:
Don't want to transition all globally - could introduce some unintended transitions